tools/bug-report/pkg/config/config.go

```diff
@@ -191,7 +191,7 @@ func (b *BugReportConfig) String() string {
 }
 
 func parseToIncludeTypeSlice(s string) []string {
-	if strings.TrimSpace(s) == "*" {
+	if strings.TrimSpace(s) == "*" || s == "" {
```

Comment:
This would initialize part of the resources (like namespace or deployment in include/exclude data  struct) to be `""`. Then this would match against anything being excluded forcing it to be skipped.